openapi_to_fastapi/validator/ihan_standards.py

```diff
@@ -82,17 +90,24 @@ def validate_spec(spec: dict):
     if not component_schemas:
         raise SchemaMissing('No "components/schemas" section defined')
 
-    if not post_route.get("requestBody"):
-        raise RequestBodyMissing
-    if not post_route["requestBody"].get("content"):
-        raise RequestBodyMissing
-    validate_component_schema(post_route["requestBody"], component_schemas)
+    if post_route.get("requestBody") and post_route["requestBody"].get("content"):
```

Comment:
Could this be simplified to just `post_route.get("requestBody", {}).get("content"):`?

Reply:
true.. missed it somehow